test/ClangImporter/windows-sdk-macros.swift

```diff
@@ -0,0 +1,10 @@
+// RUN: %swift -target i686-unknown-windows-msvc -typecheck %s -parse-stdlib -dump-clang-diagnostics 2>&1 | %FileCheck %s -check-prefix CHECK-X86
```

Comment:
Will these targets work on Darwin and Linux too or do we need to add a REQUIRES: line?

Reply:
We probably need a `REQUIRES: X86,ARM` for platforms that don't have those backends configured, but other than that this should work okay. We do similar things elsewhere to test the driver.

(I was expecting an `-import-objc-header` test but this works too.)

Reply:
@jrose-apple hmm ... this shouldn't CG anything, I *think* that this should be safe without the REQUIRES.

Reply:
The Clang importer is still set up in such a way that requires a valid CodeGen target, I think.